conda/environments/all_cuda-129_arch-aarch64.yaml

```diff
@@ -44,9 +44,9 @@ dependencies:
 - myst-parser
 - ninja
 - notebook
-- numba-cuda>=0.19.1,<0.20.0a0
+- numba-cuda>=0.22.1,<0.23.0
```

Comment:
We updated `numba-cuda` everywhere in RAPIDS but missed cuOpt. This will cause build/runtime issues in the 26.02 release if not updated.

Reply:
This was also updated in #717. 😄